notebooks/official/workbench/spark/spark_ml.ipynb

```diff
@@ -0,0 +1,1567 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.        col("pickup_location_id").cast("int").alias("start_zone_id"),
</pre><p>What do you gain by casting these as ints?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/838/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
<p>location_id is a range of numbers from 0 to 263 but it is stored as string.</p>